json.go

```diff
@@ -0,0 +1,73 @@
+package clover
+
+import (
+	"os"
+	"encoding/json"
+	"bufio"
+	"io/ioutil"
+	"errors"
+)
+
+// Exports a `collection` to JSON format in `exportPath`
+func (db *DB) ExportCollection(collectionName string, exportPath string) error{
+	exists, err := db.HasCollection(collectionName)
+	if err != nil{
+		return err
+	}
+	if !exists{
+		return ErrCollectionNotExist
+	}
+
+	_, err = os.Stat(exportPath)
+	if err == nil{
+		return os.ErrExist
+	}
+	if !errors.Is(err, os.ErrNotExist){
+		return err
+	}
+
+	result, err := db.Query(collectionName).FindAll()
+	if err != nil{
+		return err
+	}
+
+	docs := make([]map[string]interface{}, 0)
+	for _, doc := range result{
+		docs = append(docs, doc.fields)
+	}
+
+	jsonString, err := json.Marshal(docs)
+	if err != nil{
+		return err
+	}
+
+	return ioutil.WriteFile(exportPath, jsonString, os.ModePerm)
+}
+
+// Creates a new `collection` from a JSON file specified by `importPath`
+func (db *DB) ImportCollection(collectionName string, importPath string) error {
+	file, err := os.Open(importPath)
+	if errors.Is(err, os.ErrNotExist){
```

Comment:
Why should you check this error separately? This check is included in lines 53-55

Reply:
No reason at all. Already removed. Was just making a few tests to make sure that `os.Open` was returning the error `os.ErrNotExist`